src/router/pam/automations.js

```diff
@@ -266,7 +191,7 @@ export default [
           menuTitle: i18n.t('AccountBackup'),
           title: i18n.t('AccountBackupList'),
           // activeMenu: '/console/accounts/automations',
-          permissions: ['accounts.view_accountbackupautomation']
+          permissions: ['accounts.view_backupaccountautomation']
         }
       },
       {
```

Comment:
The above content is in Chinese. It seems there's no discrepancy or issue in the current implementation of the code according to provided criteria, though please note that I might not have access to real-time resources like an updated list of available APIs. If you need help with specific API calls or operations within this context, feel free to request it.